src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/DecreeAbsoluteController.java

```
@@ -0,0 +1,52 @@
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.divorce.orchestration.domain.model.ccd.CcdCallbackRequest;
+import uk.gov.hmcts.reform.divorce.orchestration.domain.model.ccd.CcdCallbackResponse;
+import uk.gov.hmcts.reform.divorce.orchestration.framework.workflow.WorkflowException;
+import uk.gov.hmcts.reform.divorce.orchestration.service.DecreeAbsoluteService;
+
+import javax.ws.rs.core.MediaType;
+
+@RestController
+@Slf4j
+@AllArgsConstructor
+public class DecreeAbsoluteController {
+
+    private final DecreeAbsoluteService decreeAbsoluteService;
+
+    @PostMapping(path = "/da-requested-respondent-notification", consumes = MediaType.APPLICATION_JSON, produces = MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Handle email notification to respondent")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "email sent.", response = CcdCallbackResponse.class),
+        @ApiResponse(code = 400, message = "Bad Request"),
+        @ApiResponse(code = 500, message = "Internal Server Error")})
+    public ResponseEntity<CcdCallbackResponse> notifyRespondentOfDARequested(
+        @RequestHeader(value = "Authorization") String authorizationToken  ,
+        @RequestBody @ApiParam("CaseData") CcdCallbackRequest ccdCallbackRequest) throws WorkflowException {
+        String caseId = ccdCallbackRequest.getCaseDetails().getCaseId();
+
+        CcdCallbackResponse.CcdCallbackResponseBuilder callbackResponseBuilder = CcdCallbackResponse.builder();
+
+        try {
+            callbackResponseBuilder.data(decreeAbsoluteService.notifyRespondentOfDARequested(ccdCallbackRequest, authorizationToken));
+            log.info("Emailed Respondent as applicant has requested DA for case {}.", caseId);
+        } catch (WorkflowException exception) {
+            log.error("Respondent notification email failed to send for applicant requesting DA. Case id: {}", caseId, exception);
+            throw exception;
```

you don't need to re-throw the exception here, you need to wrap your the error into CcdCallbackResponse .errors so on CCD screen will be able to show the error message

---

Changed to CcdCallbackResponse with error.